### PR TITLE
add Etymology of kuberpult

### PR DESCRIPTION
**Pull request by sven-urbanski-freiheit-com**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # kuberpult Readme for users
 
+## Etymology
+
+kuberpult is a catapult for [kubernetes](https://kubernetes.io/) :) it catapults the kubernetes containers to different stages (like production)
```
Comment (sven-urbanski-freiheit-com) [inline README.md:5]:
```suggestion
Kuberpult is a catapult for [kubernetes](https://kubernetes.io/) :) it catapults the kubernetes containers to different stages (like production).
```

**Review by tameremad — APPROVED** [review]
Happily Approved :D

**Review by medo-freiheit — APPROVED** [review]

**Comment by medo-freiheit** [comment]
Ok, now it makes sense to me 😄

**Review by mouhsen-ibrahim — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # kuberpult Readme for users
 
+## Etymology
+
+Kuberpult is a catapult for [kubernetes](https://kubernetes.io/) :) it catapults the kubernetes containers to different stages (like production).
```
Comment (e96wic) [inline README.md:5]: Technically not _kubernetes_ containers. Maybe rather "catapults the application containers into different stages' kubernetes clusters"

Comment (sven-urbanski-freiheit-com) [inline README.md:5]:
```suggestion
Kuberpult is a catapult for [kubernetes](https://kubernetes.io/) :) it catapults the containers of microservices to different stages (like production) in kubernetes clusters.
```

Comment (sven-urbanski-freiheit-com) [reply README.md]: I rephrased it.

**Pull request merged**